### ISSUE TITLE
icelogo standalone does not complete the analysis

### ISSUE THREAD

**Issue by 2kleio** [compomics/icelogo#5]
Hi there, 
I am trying to use the standalone version of icelogo (as the Server is not working for the last 2 weeks) on Windows 10 computer with Java version 8 update 341. I follow the manual for the Static reference method and use Homo sapiens Swiss-Prot means as the background. The software seems to start the analysis but then stop/hang without any progress and never finishes the analysis nor presents the outcome.
Can you please advise?
Many thanks

**Comment by EvelinaPillai** [comment]
Hi, 
I have the same issue with running on Windows 10. So I have run it on a VirtualMachine with Ubuntu 18.04 and openjdk-1.8.0_312 and the logo is created in seconds. I only tested it with the provided data but seems like a java version and windows issue.
So as work around you could use a Linux system if you can or you can try an older  java version 8 update 312 until someone of the developers hopefully answer. 
I also updated my Java on Ubuntu to 1.8.0_342 , it still works fine.

**Comment by 2kleio** [comment]
Thanks EvelinaPillai. I will give it a go.

**Comment by wangshisheng** [comment]
Yes, I have also the same issue on Windows 10. Hope any developer could help to solve this. Thanks~~
